app/js/components/orderEntry/OrderEntryPage.jsx

```diff
@@ -485,9 +496,6 @@ const mapStateToProps = ({
   dateFormatReducer,
   inpatientCareSetting,
   settingEncounterTypeReducer,
-  settingEncounterRoleReducer,
-  patient,
-  note,
```

Comment:
For some reason these three lines got moved to the block down below @charlesdupuyrony ... this is what was causing that error we were seeing yesterday.